src/Nowruz/pages/search/useSearch.tsx

```diff
@@ -116,12 +116,7 @@ export const useSearch = () => {
     (item: Job | Organization | User) => {
       if (type && ['users', 'organizations'].includes(type)) {
         return (
-          <div
-            onClick={e => {
-              if (e.target === e.currentTarget) handleNavigate(item as Organization | User);
```

Comment:
It seems that checking e.target is necessary. Otherwise, when user clicks on connect button on search profile result, the profile page is loaded. 'Connect' button is inside this 'div' and event bubbling happens

Reply:
We should except the connect button not the whole div 
Because this condition makes the div to not be clickable.